plone/stringinterp/adapters.py

```diff
@@ -441,35 +407,35 @@ def getPropsForIds(self, ids, propname):
         return self.getPropsForMembers(self.getMembersFromIds(ids), propname)
 
 
-# A base class for all the role->email list adapters
+# A base class for all the role-&gt;email list adapters
```

Comment:
This is not html.  Let's change this back to what it was.  Or actually make it proper text instead of a symbol.

```suggestion
# A base class for all the role to email list adapters
```